_data/logistics.yml

```diff
@@ -0,0 +1,22 @@
+-
```

Comment:
We don't need the `_data/logistics.yml` file anymore, right?

Reply:
correct - will remove it